akka-actor-tests/src/test/scala/akka/util/FrequencySketchSpec.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2021 Lightbend Inc. <https://www.lightbend.com>
+ * Copyright (C) 2021 - 2022 Lightbend Inc. <https://www.lightbend.com>
```

Comment:
We typically don't use spaces around the years:

```suggestion
 * Copyright (C) 2021-2022 Lightbend Inc. <https://www.lightbend.com>
```

It's easy to automatically update the headers by running `sbt headerCreate test:headerCreate`